README.md

```diff
@@ -8,13 +8,17 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 # BenchExec
+
 ## A Framework for Reliable Benchmarking and Resource Measurement
 
 [![Build Status](https://gitlab.com/sosy-lab/software/benchexec/badges/main/pipeline.svg)](https://gitlab.com/sosy-lab/software/benchexec/pipelines)
 [![Apache 2.0 License](https://img.shields.io/badge/license-Apache--2-brightgreen.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 [![PyPI version](https://img.shields.io/pypi/v/BenchExec.svg)](https://pypi.python.org/pypi/BenchExec)
 [![DOI](https://zenodo.org/badge/30758422.svg)](https://zenodo.org/badge/latestdoi/30758422)
 
+> [!NOTE]
+> To get started with reliably benchmarking right away, follow the
+> [quickstart guide](doc/quickstart.md).
```

Comment:
I would move this below the "News and Updates".

Also, we have to restrict this statement somehow. Right now it reads as if everyone could (and should?) just follow this link, but the tutorial targets only a part of potential BenchExec users. So we send some people the wrong way.

Besides that, using the phrase "we offer" (a quickstart guide) makes it sound more like a nice option instead of telling people to not bother with the rest of the readme.